src/transport/heat_model.cc

```diff
@@ -42,22 +42,79 @@ using namespace std;
 using namespace Input::Type;
 
 
-
-
-
-
-
+const std::map<unsigned int,unsigned int> HeatTransferModel::ModelEqData::bc_type_conversion = {
```

Comment:
For the moment it seems redundant, but this allows definition of different bc types (names) for Model classes. We could have e.g. "insulated_wall" in heat transfer or "precipitation" in solute transport.